.github/workflows/tests-polyfill.yml

```diff
@@ -1,300 +1,21 @@
 name: Polyfill Tests
 
 on:
-  push:
-    branches:
-      - main
-      - dev 
+  pull_request_target:
+    types: [labeled]
```

Comment:
We went away from running polyfill tests running in PRs.  It causes some concurrency problems to do this in PRs with the other tests running, and is overkill.  I think polyfill tests can just be ran before we cut a release.

Reply:
For trunk-based + reduction to just IE11, you'd bring it back to per PR merge into trunk?

Reply:
I brnached from the wrong brnach guys. One sec